src/base/_defaults.scss

```diff
@@ -268,3 +269,17 @@ figcaption {
 address {
   font-style: normal;
 }
+
+/**
+ * Horizontal rules
+ *
+ * We use border styles and `currentColor` for greater compatibility with
+ * the WordPress Gutenberg "Separator" block.
+ */
+
+hr {
+  border-color: currentColor;
+  border-style: solid;
+  border-width: math.div(size.$edge-medium, 2) 0;
```

Comment:
I trust you have a good reason, so I'm still going to approve, but perhaps you could add a comment here explaining why this is set to 1/2 of `$edge-medium`, rather than just using `$edge-small`?

Reply:
@spaceninja I've pushed up an additional comment that I hope explains it better.